doc/source/whatsnew/v0.24.0.rst

```diff
@@ -1114,6 +1115,7 @@ Deprecations
 - The methods :meth:`Series.str.partition` and :meth:`Series.str.rpartition` have deprecated the ``pat`` keyword in favor of ``sep`` (:issue:`22676`)
 - Deprecated the `nthreads` keyword of :func:`pandas.read_feather` in favor of
   `use_threads` to reflect the changes in pyarrow 0.11.0. (:issue:`23053`)
+- :meth:`ExtensionArray._formatting_values` is deprecated. Use `ExtensionArray._formatter` instead. (:issue:`23601`)
```

Comment:
can you add the ``:meth:`` syntax for ``_formatter`` (and maybe remove for ``_formatting_values``. is this removed from ``api.rst`` (for _formatter_values) and added for ``_formatter``)?

Reply:
Fixing the `:meth:`. There's a problem with api.rst excluding "private" methods: https://github.com/pandas-dev/pandas/issues/24067